app/views/organisations/index.html.erb

```diff
@@ -3,15 +3,11 @@
 
 <% content_for :title, title %>
 
-<h1 class="govuk-heading-l">
-  <% if current_user.support? && request.path == "/organisations" %>
-    Organisations
-  <% end %>
-  <% if current_user.support? && request.path != "/organisations" %>
-    <span class="govuk-caption-l"><%= @organisation.name %></span>
-    <h1 class="govuk-heading-l">Organisations</h1>
-  <% end %>
-</h1>
+<% if request.path == "/organisations" %>
+  <%= render partial: "headings", locals: { main: "Organisations", sub: nil } %>
+<% else %>
+  <%= render partial: "headings", locals: { main: @organisation.name, sub: "Organisations" } %>
+<% end %>
```

Comment:
You could just set locals in the if/else since you're always rendering the same partial in both cases?

Reply:
Ture, will look into this. Thought about moving logic into partial, but it has quite a few conditions and it won't pretty or easy to digest or scale.